packages/core/src/button/inline-button.element.ts

```diff
@@ -29,6 +29,8 @@ import styles from './inline-button.element.scss';
  * @cssprop --letter-spacing
  */
 export class CdsInlineButton extends CdsBaseButton {
+  @querySlot('cds-icon') protected icon: HTMLElement;
```

Comment:
Icon is no longer part of the base button class as only a few actually used it and created false api docs